unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/Tests/BackpackInfoCardViewTest.cs

```diff
@@ -0,0 +1,47 @@
+using NUnit.Framework;
+using UnityEngine;
+
+namespace DCL.Backpack
+{
+    public class BackpackInfoCardViewTest
```

Comment:
its missing to cover many cases in this test like rarity, hide list, remove list, hidden by list, etc..

Reply:
yeah because the integration might vary a lot between now and when we merge all together, I'll add them in a second phase